docs/docs/genai/concepts/scorers.mdx

```diff
@@ -0,0 +1,230 @@
+import { APILink } from "@site/src/components/APILink";
+import ImageBox from "@site/src/components/ImageBox";
+import TabsWrapper from "@site/src/components/TabsWrapper";
+import FeatureHighlights from "@site/src/components/FeatureHighlights";
+import ConceptOverview from "@site/src/components/ConceptOverview";
+import TilesGrid from "@site/src/components/TilesGrid";
+import TileCard from "@site/src/components/TileCard";
+import { Brain, Code, Users, Shield, Target, Activity, ChartBar, AlertTriangle, GitBranch } from "lucide-react";
+
+# Scorer Concepts
+
+## What are Scorers?
+
+**Scorers** in MLflow are evaluation functions that assess the quality of your GenAI application outputs. They provide a systematic way to measure performance across different dimensions like correctness, relevance, safety, and adherence to guidelines.
+
+Scorers transform subjective quality assessments into measurable metrics, enabling you to track performance, compare models, and ensure your applications meet quality standards. They range from simple rule-based checks to sophisticated LLM judges that can evaluate nuanced aspects of language generation.
+
+## Use Cases
+
+<FeatureHighlights features={[
+  {
+    icon: Brain,
+    title: "Automated Quality Assessment",
+    description: "Replace manual review processes with automated scoring that can evaluate thousands of outputs consistently and at scale, using either deterministic rules or LLM-based evaluation."
+  },
+  {
+    icon: Shield,
+    title: "Safety & Compliance Validation",
+    description: "Systematically check for harmful content, bias, PII leakage, and regulatory compliance. Ensure your applications meet organizational and legal standards before deployment."
+  },
+  {
+    icon: GitBranch,
+    title: "A/B Testing & Model Comparison",
+    description: "Compare different models, prompts, or configurations using consistent evaluation criteria. Make data-driven decisions about which approach performs best for your use case."
+  },
+  {
+    icon: ChartBar,
+    title: "Continuous Quality Monitoring",
+    description: "Track quality metrics over time in production, detect degradations early, and maintain high standards as your application evolves and scales."
+  }
+]} />
+
+## Types of Scorers
+
+MLflow provides several types of scorers to address different evaluation needs:
+
+<ConceptOverview concepts={[
+  {
+    icon: Brain,
+    title: "LLM-based Scorers (LLM-as-a-Judge)",
+    description: "Use large language models to evaluate subjective qualities like helpfulness, coherence, and style. These scorers can understand context and nuance that rule-based systems miss."
+  },
+  {
+    icon: Code,
+    title: "Code-based Scorers",
+    description: "Custom Python functions for deterministic evaluation. Perfect for metrics that can be calculated algorithmically like ROUGE scores, exact match, or custom business logic."
+  },
+  {
+    icon: Activity,
+    title: "Agentic Judges",
```

Comment:
```suggestion
    title: "Agent-based Scorers (Agent-as-a-judge)",
```

@BenWilson2 Can we audit all the docs and make sure we say "agent as a judge" instead of agentic?

Also, can we move this above the code-based scorers?